test/expected/bgw_launcher.out

```diff
@@ -435,14 +489,41 @@ SELECT wait_worker_counts(1,1,0,0);
  t
 (1 row)
 
--- clean up additional database
-\c :TEST_DBNAME :ROLE_SUPERUSER
-DROP DATABASE :TEST_DBNAME_2;
 -- test rename database
 CREATE DATABASE db_rename_test;
 \c db_rename_test :ROLE_SUPERUSER
 SET client_min_messages=error;
 CREATE EXTENSION timescaledb;
-\c :TEST_DBNAME :ROLE_SUPERUSER
+\c :TEST_DBNAME_2 :ROLE_SUPERUSER
+SELECT wait_for_bgw_scheduler('db_rename_test');
```

Comment:
don't we want this to run after the rename? Or at least both before and after?

Reply:
we wanna test that rename works when there is a background worker active so we need to be sure the background worker has been started, testing it after would test that the restart actually worked which is not what this pr is about but could be tested here aswell